app/models/vim_performance_daily.rb

```diff
@@ -25,7 +25,7 @@ def self.excluded_cols_for_expressions
   def self.find(cnt, *args)
     raise "Unsupported finder value #{cnt}" unless cnt == :all
 
-    Vmdb::Deprecation.deprecation_warning(:find, :find_entries)
+    Vmdb::Deprecation.deprecation_warning(:find, :find_entries, caller(1))
```

Comment:
@matthewd I tried looking at the activesupport code and I don't see how to fix this without removing the delegation.  :confused:  It's so hard to follow the deprecation code.